pkg/didcomm/protocol/mediator/models.go

```diff
@@ -38,7 +38,10 @@ type KeylistUpdate struct {
 // Update route key update message.
 type Update struct {
 	RecipientKey string `json:"recipient_key,omitempty"`
+	RecipientKID string `json:"recipient_kid,omitempty"`
```

Comment:
this would require updates to Aries RFCs, if really needed

Reply:
indeed.. which triggered me to discuss this on the A WG meeting last week. The same goes with EncKey and EncKID. All these new fields are unused in this patch. I will remove them for now and probably add them once the RFCs are updated.

Reply:
They're now removed, we'll deal with full Encryption keys in RecipientKeys (not just Ed25119) in a future PR once didComm/DID Exchange RFCS are updated.